crates/aggchain-proof-core/src/bridge/mod.rs

```diff
@@ -670,9 +674,21 @@ mod tests {
                 .expect("Invalid URL format");
 
             let evm_sketch = |block_number: u64| {
+                // Remove comment lines from the JSON string before parsing
+                let json_clean: String = CUSTOM_JSON
+                    .lines()
+                    .filter(|line| !line.trim_start().starts_with("//"))
+                    .collect::<Vec<_>>()
+                    .join("\n");
+
+                // Parse the JSON into alloy_genesis::Genesis first, then extract the config
+                let genesis_parsed: alloy::genesis::Genesis =
+                    serde_json::from_str(&json_clean).expect("Failed to parse genesis JSON");
+
```

Comment:
Could the function `aggchain_proof_contracts::config::parse_evm_sketch_genesis` be used here? It  could be fixed/updated if needed.